client/landing/stepper/declarative-flow/internals/steps-repository/link-in-bio-post-setup/index.tsx

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+import { StepContainer, base64ImageToBlob, uploadAndSetSiteLogo } from '@automattic/onboarding';
```

Comment:
Slightly lost about `uploadAndSetSiteLogo` and D87847-code.

If `uploadAndSetSiteLogo` existed before this, how did it work without that diff?  If it worked without that diff, why did we need to add it?

Reply:
@Addison-Stavlo @alshakero - It occurred to me to freshly retest this to confirm things. It is easy enough to do by running this branch of calypso without the associated backend diff. As it turns out this is working for me now without the backend diff. I'm honestly not sure what I may have been doing differently in development to get a different result. And I'd say that the logic of the endpoint suggests this shouldn't work (it means that the settings endpoint is ignoring both the request_format and the whitelisting filter, which raises the question of why those are there). 

In any case, I think that means we can adopt this calypso PR without the associated backend diff.